tutorials/ui/size_and_anchors.rst

```diff
@@ -60,6 +61,25 @@ a TextureRect can be centered in its parent:
     rect.margin_bottom = -texture_size.y / 2
     add_child(rect)
 
+ .. code-tab:: csharp
+     
+    var rect = new TextureRect();
+    
+    rect.Texture = GD.Load("res://icon.png");
+    rect.AnchorLeft = 0.5f;
+    rect.AnchorRight = 0.5f;
+    rect.AnchorTop = 0.5f;
+    rect.AnchorBottom = 0.5f;
+     
+    var TextureSize = rect.Texture.GetSize();
```

Comment:
```suggestion
    var textureSize = rect.Texture.GetSize();
```
(and same change below).

Please make sure to follow the naming conventions from the C# style guide: https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/c_sharp_style_guide.html#naming-conventions

Reply:
```suggestion
    Vector2 textureSize = rect.Texture.GetSize();
```

Also, because there's no type indication on the right-hand side, we'd prefer to declare explicitly a `Vector2`.